include/CppUTestExt/MockNamedValue.h

```diff
@@ -40,6 +40,7 @@ class MockNamedValueComparator
 
     virtual bool isEqual(const void* object1, const void* object2)=0;
     virtual SimpleString valueToString(const void* object)=0;
+    virtual void* copy(const void*, const void*) { return 0; }
```

Comment:
Why return anything at all if the returned value is not used? 

`void` return value could be a better choice, or maybe return `bool` and check in `MockCheckedActualCall::finalizeOutputParameters` that the value is not `false`(and throw an exception / error otherwise), such that the default implementation returns `false` and overriden implementatations should return `true`. That way if a user installs a comparator without overriding `copy` and it's used to copy an output parameter, it doesn't fail silently (which can be hard to detect).